web/cypress/integration/roadmap-mobile.spec.ts

```diff
@@ -22,6 +22,10 @@ describe("Roadmap [feature] [all]", () => {
     cy.get('[data-municipality="Absecon"]').should("exist");
 
     // step 1
+    cy.get('[id="plan-content"]').should("be.visible");
+    cy.get('[id="plan-header"]').click({ force: true });
+    cy.get('[id="plan-content"]').should("not.be.visible");
+    cy.get('[id="plan-header"]').click({ force: true });
```

Comment:
is there a reason for using a proper ID here instead of a data-testid?

a proper ID would be necessary if we are using aria-labelledby or something, I think. Or, if we are using specific CSS for those IDs (although, that would be a yellow flag to ask why...)

Whereas if we're only using it as a selector for testing, `data-testid` is a better choice because it is explicitly saying "hey this ID is not semantically / functionally meaningful, it is just for testing selector purposes"

Reply:
@aloverso Per MUI Doc, for optimal accessibility they recommend setting ```id``` and ```aria-controls``` on the ```AccordionSummary```. The Accordion will derive the necessary ```aria-labelledby``` and ```id``` for the content region of the accordion.

Reply:
Make sense to me